src/app/home/pages/PlansPage/components/Wizard/MigrationOptions/MigrationOptionsForm.tsx

```diff
@@ -45,12 +45,16 @@ const MigrationOptionsForm: React.FunctionComponent<IMigrationOptionsFormProps>
 
   useEffect(() => {
     //if available, set direct migration for image/volumes on initial render
-    if (isDirectImageMigrationAvailable) {
+    if (isDirectImageMigrationAvailable && !isEdit) {
```

Comment:
Added !isEdit flag to fix plan spec user selection being overwritten when editing. Logic reads "If initial creation of plan & direct migration is available, check the direct migration checkbox. If initial creation of plan and direct migration is not available, do not select the checkbox".